conan/tools/env/virtualbuildenv.py

```diff
@@ -8,9 +8,10 @@ class VirtualBuildEnv:
         .bat or .sh script
     """
 
-    def __init__(self, conanfile):
+    def __init__(self, conanfile, skip_auto_generate=True):
```

Comment:
I was thinking in other alternatives for the name of the argument, maybe one of these or any variation, makes sense:

- `freeze`
- `hold`
- `no_refresh`
- `keep`
- `inmutable`

Anyway, I think that `skip_auto_generate` or just `auto_generate` are also good enough

Reply:
Lets do ``auto_generate``, I think it would be the most (relatively speaking) intuitive.